server.js

```diff
@@ -1,33 +1,72 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
-
-// Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
-
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import netflixData from "./data/netflix-titles.json";
+
+const port = process.env.PORT || 8080;
+const app = express();
+
+// Middle wares to enable cors and json body parsing
+app.use(cors());
+app.use(bodyParser.json());
+
+app.get("/", (req, res) => {
+  res.send(
+    "My API endpoints: /all , all/comedies/ , /all:id , all/star?name=Brad%20Pitt ,  /movie, /tv, /year/:year , plus  handling error if no id match"
+  );
+});
+
+// Get ALL movies and tv shows:
+app.get("/all", (req, res) => {
+  res.json(netflixData);
+});
+
+// Get all by actor
+app.get("/all/star", (req, res) => {
+  const { name } = req.query;
+  const movieStar = netflixData.filter((person) => person.cast.includes(name));
+  res.json(movieStar);
+});
```

Comment:
A good alternative is to include this functionality as a query parameter in `/all?actor=Pitt`